libleptongui/src/x_event.c

```diff
@@ -893,7 +901,15 @@ x_event_faked_motion (GschemPageView *view, GdkEventKey *event) {
   gboolean ret;
   GdkEventMotion *newevent;
 
+#ifdef ENABLE_GTK3
+  GdkDisplay *display = gdk_display_get_default ();
+  GdkSeat *seat = gdk_display_get_default_seat (display);
+  GdkDevice *pointer = gdk_seat_get_pointer (seat);
+
+  gdk_device_get_position (pointer, NULL, &x, &y);
```

Comment:
Using ` gdk_device_get_position()` here has a strange effect:
please try to open add comp. dialog, select some symbol,
place cursor over the main window and do zoom in/out
with the mouse wheel.
Do you see the symbol jumping up and down when zooming, like I do?

Reply:
Yes, it looks like it jumps a bit.  Maybe the issue with `GDK_SCROLL_SMOOTH` support I've already presented before.  I did some work on it before though cannot recall in which state I left it.  Need to re-check.